server/services/lists/comCaseloadService.ts

```diff
@@ -185,8 +215,11 @@ export default class ComCaseloadService {
               id: licence.licenceId,
               status: licence.isReviewNeeded ? LicenceStatus.REVIEW_NEEDED : <LicenceStatus>licence.licenceStatus,
               type: <LicenceType>licence.licenceType,
-              comUsername: licence.comUsername,
               kind: <LicenceKind>licence.kind,
+              crn: licence.crn,
+              nomisId: licence.nomisId,
+              name: convertToTitleCase(`${offender.nomisRecord.firstName} ${offender.nomisRecord.lastName}`.trim()),
```

Comment:
**nitpick (non-blocking):** I think we can also take name from the licence, but don't worry about it if you don't need to make any other code changes

Reply:
updated